Laserfiche/lf-repository-api-client-js
# Update with latest swagger and update documentation (changelog, migration guide, function docs)

- Use the latest swagger schema to regenerate the client
- In each manually created forEach and nextlink functions, add a link to the auto generated function. Rather than copying documentation from the auto generated function like required oauth scope and conditions for the api call, it would be easier to just add a link to the other function for now.
- Update the nextlink `@returns` documentation to match the new format in the auto generated functions
- Restart the changelog in preparation for the new package release
- Add migration guide to link the v1 api functions to the v2 api functions. Currently the v2 links don't work, but they should work after the v2 documentation has been published
  - https://github.com/Laserfiche/lf-repository-api-client-js/blob/update-documentation/MIGRATION_GUIDE.md
